app/src/main/java/org/oppia/android/app/drawer/ExitProfileDialogInterface.kt

```diff
@@ -2,5 +2,6 @@ package org.oppia.android.app.drawer
 
 /** Interface to handle option selection in [ExitProfileDialogFragment]. */
 interface ExitProfileDialogInterface {
-  fun markHomeMenuCloseDrawer()
+  fun markLastCheckedItemCloseDrawer(lastCheckedItemId: Int, isAdminSelected: Boolean)
```

Comment:
same here `isAdministratorControlsSelected ` and other places too.

Reply:
Done